tutorial/tutorial-template-guide.md

```diff
@@ -0,0 +1,132 @@
+
+### About the "Before you begin" section
+
+Use this section to list any prerequisites for completing the tutorial. This may include operating systems, languages, package managers, or software.
+
+### About the "Steps" section
+
+The steps section is where you describe what the user needs to do to complete the tutorial. The template organizes steps in the following way:
+
+```
+1. {Write the command here. Start with a verb.}
+
+   {Optional: Explanatory text}
+
+   {Optional: Code sample or screenshot that helps your users complete this step}
+
+   {Optional: Result}
+
+```
+
+The first element is the step itself, like "Enter a name for your new repository." When writing steps, keep these tips in mind:
+
+* Start with an imperative verb. For example, "connect", "set up", or "arrange." Don't use the *-ing* form of the verb because it is harder to translate.
+* Express steps as a complete thought. Instead of saying, "Set up access", you might say, "Set up access to a Cloud Storage bucket."
+
+Next, you may choose to add some additional information about the step. Continuing with the example, you might provide some best practices for creating memorable repository names.
+
+Optionally, you can add a code sample or screenshot after the explanatory text, depending on the type of tutorial you're writing. Screenshots are a great way to show specific parts of the screen you are referring to in a step. 
+
+If you're including code samples in your repository, make sure to add comments to your code to help learners understand what each part of the code does. Additionally, make sure your code samples work, as learners will copy and paste your code to use in their own projects.
+
+The last, and optional, element in a step, is the result. Continuing with the example, you might describe what happens after a user click the "Create repository" button.
+
+Here is an example step:
+
+```
+1. Enter a name for your new repository.   
+
+   Good repository names are short and self-explanatory. Avoid repository names with three or more words.
+   
+   After you click "Create repository", GitHub creates your repository and the main page for the repository is displayed.
+```
+
+If needed, you can add substeps below a primary step. Make sure to indent the substep one tab space over. 
+
+```
+1. Create a new pull request.
+
+   a. Navigate to the main page of your repository. 
+
+   b. Under your repository name, click **Pull requests**.
+
+      By default, all open pull requests are displayed.
+
+   c. ....
+```
+
+#### Tips for writing steps
+
+Here are some more tips to follow when writing procedural steps:
+
+* Remember to orient your users when walking them through each step. If they need to open a particular file or dialog to complete the task, provide that information first.
+* Avoid writing procedures that are more than seven primary steps long.
+* Aim for no more than four substeps in any primary step.
```

Comment:
**Comment:** Hmmm. I think I break the four substeps rule a lot. I'm not sure if that's because it's me not following best practices or if I disagree with this advice. I'd have to think about it.

Reply:
@barbaricyawps Thinking about it, me too. I pulled this guidance from a link Cameron provided. I might cap substeps at 6, maybe 7? 

I could revise the advice to six or fewer.